frontend/src/services/fetchData.ts

```diff
@@ -66,3 +69,41 @@ export const fetchApplicationRoles = async (
     ).data;
     return applicationRoles;
 };
+
+export const fetchApplicationAdmins = async (
+    applicationId: number | undefined
+): Promise<FamAppAdminGet[]> => {
+    if (!applicationId) return [];
+    const applicationAdmin = (
+        await AdminMgmtApiService.applicationAdminApi.getApplicationAdminByApplicationId (
```

Comment:
So fetch application admins should return all application admins, not only FAM. When user select FAM from the dropdown, it will list the admins for all apps, cause FAM admin can manage the admin for all applications. @ianliuwk1019 I feel we need to add a new backend api call to return admins for all applications.

Reply:
Correct @MCatherine1994 ; the new function `fetchApplicationAdmins` shouldn't switch context to call `getApplicationAdminByApplicationId` from API during implementation.
We can either change this `getApplicationAdminByApplicationId` (since for now don't seem to have a need for this function) on backend or create a new api at backend. I can have a quick adjustment at the backend (in separate PR) for incorporating the missing function tomorrow if needed urgently, but we probably need to discuss with Olga. This PR cannot be released alone without the change.

Reply:
yeah we have to do the backend api first. Let's chat with Olga, this ticket needs that api.